File: programmers/level2/pro42576.py
def solution(participant, completion):
    answer = ''
    temp = {}
    for name in participant:
        if(temp.get(name) != None):
            temp[name] += 1
        else : temp[name] = 0
    for name in completion:
        if(temp[name] == 0):
            temp.pop(name)
        else :
            temp[name] -= 1
    for i in temp.keys():
        answer = i
    return answer

File: javascript/pro42576.js
function solution(participant, completion) {
    const complete = {};
    let answer = '';
    participant.forEach(person => {
        if (complete[person]) {
            complete[person] += 1;
        }
        else {
            complete[person] = 1;
        }
    });
    completion.forEach(person => {
        complete[person] -= 1;
    });
    Object.keys(complete).forEach(person => {
        if (complete[person] > 0) {
            answer = person;
        }
    })
    return answer;
}

let p = ["leo","kiki", "eden"];
let complete = ["eden", "kiki"];
console.log(solution(p, complete));